vmdb/spec/lib/miq_automation_engine/miq_ae_state_machine_new_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+require 'timecop'
+
+describe "MiqAeStateMachine" do
+  before do
+    DummyWorkspace = Class.new do
+      def initialize(options = {})
```

Comment:
Except don't put TestClass at the global namespace like that because it pollutes other tests.  There are other tests that use a TestClass as well.  @kbrock's suggestion of Class.new prevents that clash from ever happening and it generally preferred.  (On that note, we need to change the other specs to do Class.new as well.)